honeybadgerbft/core/binaryagreement.py

```diff
@@ -1,10 +1,18 @@
 import gevent
 from gevent.event import Event
+
 from collections import defaultdict
+from distutils.util import strtobool
+from os import environ
+import logging
 
 from honeybadgerbft.exceptions import RedundantMessageError, AbandonedNodeError
 
 
+logger = logging.getLogger(__name__)
+CONF_PHASE = strtobool(environ.get('CONF_PHASE', '1'))
```

Comment:
Are environment variables documented anywhere?

Reply:
Is there any reason to disable the conf phase except for testing?

Reply:
I'll remove the switch. It was put there temporarily to ease the demonstration of the attack.

By putting the `CONF` phase logic in a function we could also simply mock that function in order to demonstrate the attack.